README.md

```diff
@@ -68,6 +68,23 @@ closure_repositories()
 You are not required to install the Closure Tools or PhantomJS. They will be
 fetched automatically.
 
+### Overriding Dependency Versions
+
+To override the version of any dependency, modify your `WORKSPACE` file to pass`omit_<dependency_name>=True` to `closure_repositories()`. Next define your
+custom dependency version. A full list of dependencies is available from
+[repositories.bzl](https://github.com/bazelbuild/rules_closure/tree/master/closure/repositories.bzl).
```

Comment:
I really like the fact that you actually linked to the file. That makes life so much easier for the reader.